packs/data/strength-of-thousands-bestiary.db/abendego-brute.json

```diff
@@ -484,12 +483,32 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>Abendego brutes deal an extra [[/r {2d8}]]{2d8 damage} with tridents, and a trident gains the shove trait while the brute is using it in melee (both the extra damage dice and the shove trait are included in the <em>fighter's fork </em>Strikes above).</p>"
+                    "value": "<p>Abendego brutes deal an extra 2d8 damage with tridents, and a trident gains the shove trait while the brute is using it in melee (both the extra damage dice and the shove trait are included in the <em>fighter's fork</em> Strikes above).</p>"
                 },
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "diceNumber": 2,
+                        "dieSize": "d8",
+                        "key": "DamageDice",
+                        "predicate": [
+                            "item:base:trident"
+                        ],
+                        "selector": "strike-damage"
+                    },
+                    {
+                        "definition": [
+                            "item:base:trident",
+                            "item:melee"
```

Comment:
```suggestion
```

Reply:
The description says "... a trident gains the shove trait while the brute is using it in melee..."
Is the melee predicate superfluous here?

Reply:
The shove trait isn't usable at range even when a weapon always has it